package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "terriajs-server",
-  "version": "3.3.1",
+  "version": "3.3.1-alpha.3",
```

Comment:
Should we increment major or minor version for this?

Reply:
I think this probably warrants a major depending on how the rest of it plays out, atm it is ok for minor given it's progressively added - maybe we need a next branch for this current piece of work. What do you think?